themes/bootstrap3/templates/holds/edit.phtml

```diff
@@ -0,0 +1,97 @@
+?>
+<h2><?=$this->transEsc('hold_edit_title')?></h2>
+<?php if ($this->helpText): ?>
+<p class="helptext"><?=$this->helpText?></p>
+<?php endif; ?>
+
+<form action="<?=$this->url('holds-edit')?>" class="form-edit-holds" method="post" name="editHolds">
+  <?=$this->flashmessages()?>
+
+  <?php foreach ($this->selectedIDS as $id): ?>
+    <input type="hidden" name="selectedIDS[]" value="<?=$this->escapeHtmlAttr($id)?>">
+  <?php endforeach; ?>
+
+  <?php if (in_array('frozen', $this->fields)): ?>
+    <div class="form-group hold-frozen">
+      <label for="frozen" class="control-label"><?=$this->transEsc('hold_edit_frozen')?>:</label>
+      <select id="frozen" name="gatheredDetails[frozen]" class="form-control">
+        <?php $current = $gatheredDetails['frozen'] ?? '';?>
+        <option value=""<?=$current === '' ? ' selected' : ''?>><?=$this->transEsc('hold_edit_no_change')?></option>
+        <option value="1"<?=$current === '1' ? ' selected' : ''?>><?=$this->transEsc('hold_edit_frozen_set')?></option>
+        <option value="0"<?=$current === '0' ? ' selected' : ''?>><?=$this->transEsc('hold_edit_frozen_unset')?></option>
+      </select>
+    </div>
+    <?php if (in_array('frozenUntil', $this->fields)): ?>
+      <div class="form-group hold-frozen-until">
+        <label for="frozen_until" class="control-label"><?=$this->transEsc('hold_edit_frozen_until')?>:</label>
+        <input id="frozen_until" type="text" name="gatheredDetails[frozenUntil]" value="<?=$this->gatheredDetails['frozenUntil'] ?? ''?>" size="10" class="form-control"/>
+        (<?=$this->dateTime()->getDisplayDateFormat()?>)
+      </div>
+    <?php endif; ?>
+  <?php endif; ?>
+
+  <?php if (in_array('startDate', $this->fields)): ?>
+    <div class="form-group hold-start-date">
+      <label for="start_date" class="control-label"><?=$this->transEsc('hold_start_date')?>:</label>
+      <input id="start_date" type="text" name="gatheredDetails[startDate]" value="<?=$this->gatheredDetails['startDate'] ?? ''?>" size="10" class="form-control"/>
+      (<?=$this->dateTime()->getDisplayDateFormat()?>)
+    </div>
+  <?php endif; ?>
+
+  <?php if (in_array("requiredByDate", $this->fields)): ?>
+    <div class="form-group hold-required-by">
+      <label for="required_by_date" class="control-label"><?=$this->transEsc('hold_required_by')?>:</label>
+      <input id="required_by_date" type="text" name="gatheredDetails[requiredBy]" value="<?=(isset($this->gatheredDetails['requiredBy']) && !empty($this->gatheredDetails['requiredBy'])) ? $this->escapeHtmlAttr($this->gatheredDetails['requiredBy']) : ''?>" size="10" class="form-control"/>
+      (<?=$this->dateTime()->getDisplayDateFormat()?>)
+    </div>
+  <?php endif; ?>
+
+  <?php if (in_array('pickUpLocation', $this->fields) && $this->pickupLocations): ?>
+    <?php
+      if (isset($this->gatheredDetails['pickUpLocation']) && $this->gatheredDetails['pickUpLocation'] !== "") {
```

Comment:
You could use `($this->gatheredDetails['pickUpLocation'] ?? '')` to eliminate the isset check...

Reply:
It was actually possible to simplify further since it doesn't need to handle all the options that the hold creation form does.